opendevin/agent.py

```diff
@@ -1,40 +1,9 @@
 from abc import ABC, abstractmethod
-from typing import List, Dict, Type
-from dataclasses import dataclass
-from enum import Enum
+from typing import List, Dict, Type, TYPE_CHECKING
 
-from .lib.event import Event
-from .lib.command_manager import CommandManager
-
-class Role(Enum):
-    SYSTEM = "system"  # system message for LLM
-    USER = "user"  # the user
-    ASSISTANT = "assistant"  # the agent
-    ENVIRONMENT = "environment"  # the environment (e.g., bash shell, web browser, etc.)
-
-@dataclass
-class Message:
-    """
-    This data class represents a message sent by an agent to another agent or user.
-    """
-
-    role: Role
-    content: str
-    # TODO: add more fields as needed
-
-    def to_dict(self) -> Dict:
-        """
-        Converts the message to a dictionary (OpenAI chat-completion format).
-
-        Returns:
-        - message (Dict): A dictionary representation of the message.
-        """
-        role = self.role.value
-        content = self.content
-        if self.role == Role.ENVIRONMENT:
-            content = f"Environment Observation:\n{content}"
-            role = "user"  # treat environment messages as user messages
-        return {"role": role, "content": content}
```

Comment:
These `Role` and `Message` are never used by langchains agent so i just remove it for simplicity.